curator-client/src/main/java/org/apache/curator/ConnectionState.java

```diff
@@ -18,7 +18,6 @@
  */
```

Comment:
Man - this class has become nearly incomprehensible. This really needs to be addressed. I'm thinking about it.

Reply:
I agree and the first time I notice this `ConnectionState` with `curator-framwork`'s `ConnectionState`, it really confused me. But we can separated the refactor or anything on this `ConnectionState` in another pass IMO.